src/handlers/teamChatHandler.js

```diff
@@ -0,0 +1,22 @@
+
+const Client = require('../../index.js');
+const DiscordTools = require('../discordTools/discordTools.js');
+const { MessageEmbed } = require("discord.js");
+
+module.exports = function (rustplus, client, message) {
+    let instance = client.readInstanceFile(rustplus.guildId);
+    let channel = DiscordTools.getTextChannelById(rustplus.guildId, instance.channelId.teamchat);
+    if (message.message.startsWith('rustPlusPlus |')) return;
```

Comment:
I've added this in previous commit:
this.trademarkString = 'rustPlusPlus | ';
Replace the string with rustplus.trademarkString